pyro/contrib/epidemiology/compartmental.py

```diff
@@ -73,9 +73,11 @@ def transition_bwd(self, params, prev, curr, t): ...
     :param list compartments: A list of strings of compartment names.
     :param int duration:
     :param int population:
+    :param int num_quant_bins: The number of quantization bins to use. Note that
+        computational cost is exponential in `num_quant_bins`. Defaults to 4.
     """
 
-    def __init__(self, compartments, duration, population):
+    def __init__(self, compartments, duration, population, num_quant_bins=4):
```

Comment:
Can you add a `*,` separator to force use as a kwarg rather than arg?
```py
def __init__(self, compartments, duration, population, *,
             num_quant_bins=4):
```

Reply:
done